src/Workspaces/Core/Portable/FindSymbols/SyntaxTree/SyntaxTreeIndex.ContextInfo.cs

```diff
@@ -151,6 +158,7 @@ private enum ContainingNodes
                 ContainsIndexerMemberCref = 1 << 7,
                 ContainsDeconstruction = 1 << 8,
                 ContainsAwait = 1 << 9,
+                ContainsTupleExpressionOrTupleType = 1 << 10,
```

Comment:
This list is out-of-date (I've just changed it in https://github.com/dotnet/roslyn/pull/28230, which was merged earlier this week).

Reply:
Never mind, I'd added the `ContainsAwait`. I'm distracted... :-S

Reply:
Ah ok :)   Thanks!